### PR TITLE
Update nsp SA customize. Skip Grype and LimitRange

### PR DESCRIPTION
**Pull request by odinnordico**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

Changes in this PR are for TAP 1.6

### CODE REVIEW

File: namespace-provisioner/customize-installation.hbs.md [hunk 1]
```diff
@@ -347,6 +349,8 @@ Options if using GitOps
       url: https://github.com/vmware-tanzu/application-accelerator-samples.git
   ```
 
+  In case a path is not specified in the additional sources configuration, the namespace provisioner will automatically generate a path as follows: `_ytt_lib/applicaton-accelerator-samples-git-ns-provisioner-samples-testing-scaning-supplychain-0`
```
Comment (anibmurthy) [inline namespace-provisioner/customize-installation.hbs.md:352]: typo: `_ytt_lib/applicaton-accelerator-samples-git-ns-provisioner-samples-testing-scaning-supplychain-0` 
`scanning`*

File: namespace-provisioner/use-case4.hbs.md [hunk 2]
```diff
@@ -5,66 +5,42 @@ work with private Git repositories.
 
 ## Disable Grype install
 
-Namespace Provisioner creates Grype scanner install as one of the [default resources](reference.md#default-resources). If you want to use another scanner for namespaces instead of Grype, disable
-Grype scanner as follows:
+If you prefer to utilize a different scanner for namespaces instead of Grype, you have the option to deactivate the installation of the default Grype scanner, which is automatically created by the Namespace Provisioner. To do so, you can follow these steps:
 
-1. Create an overlay secret as follows which removes the Grype scanner and the secret that is automatically created by Namespace Provisioner.
+### Disable for all namespaces
```
Comment (anibmurthy) [inline namespace-provisioner/use-case4.hbs.md:10]: You might get `woke` error. Would you want to call it `Deactivate`?

Comment (odinnordico) [reply namespace-provisioner/use-case4.hbs.md]: Fixed woke findings

File: namespace-provisioner/customize-installation.hbs.md (hunk 1 above)
Comment (odinnordico) [reply namespace-provisioner/customize-installation.hbs.md]: Fixed

**Review by anibmurthy — APPROVED** [review]

**Comment by kcoriordan** [comment]
Is this ready to merge?

**Pull request merged**